core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -394,6 +394,9 @@ query.timeout.sec=60
 
 ################ Parallel SQL execution ################
 
```

Comment:
I think we should mention here that jit won't work unless  parallel filter is enabled .

Reply:
Expanded the comment in 5fb86a0e03b358a46a449c8f81e584e40a45af2e